src/builtins.c

```diff
@@ -1583,11 +1582,197 @@ DLLEXPORT void jl_(void *jl_value)
     in_jl_--;
 }
 
+// Useful because the jl_typeof macro isn't available from debugger
```

Comment:
i found out today that if you type `macro define jl_typeof jl_typeof` then it works in gdb (it already works in lldb).